Task heat_demand_timeseries.district-heating fails

The task heat_demand_timeseries.district-heating fails sometimes with the error message: 
`AssertionError: Deviation of residential heat demand time 
            series for district heating grid 2101 is 0.10490490667886568`
It is proposed to increase the acceptable tolerance in total deviation to 11%, and through a warning message when it is bigger than 3%.

--- a/src/egon/data/datasets/heat_demand_timeseries/__init__.py
+++ b/src/egon/data/datasets/heat_demand_timeseries/__init__.py
@@ -1,6 +1,7 @@
 from datetime import datetime
 from pathlib import Path
 import os
+import warnings
 
 from sqlalchemy import ARRAY, Column, Float, Integer, String, Text
 from sqlalchemy.ext.declarative import declarative_base
@@ -362,9 +363,13 @@ def create_district_heating_profile_python_like(scenario="eGon2035"):
             )
 
             assert (
-                abs(diff) < 0.03
+                abs(diff) < 0.11
             ), f"""Deviation of residential heat demand time 
             series for district heating grid {str(area)} is {diff}"""
+
+            if abs(diff) > 0.03:
+                warnings.warn(f"""Deviation of residential heat demand time
+                series for district heating grid {str(area)} is {diff}""")
 
             hh = np.concatenate(
                 slice_df.drop(

--- a/src/egon/data/datasets/heat_demand_timeseries/__init__.py
+++ b/src/egon/data/datasets/heat_demand_timeseries/__init__.py
@@ -1075,7 +1075,7 @@ class HeatTimeSeries(Dataset):
     def __init__(self, dependencies):
         super().__init__(
             name="HeatTimeSeries",
-            version="0.0.7",
+            version="0.0.8",
             dependencies=dependencies,
             tasks=(
                 {

